ccd-definitions/src/main/java/uk/gov/hmcts/reform/divorce/ccd/event/DraftCreate.java

```diff
@@ -18,25 +18,37 @@
 public class DraftCreate implements CcdBuilder {
 
     @Override
-    public void buildWith(final ConfigBuilder<CaseData, State, UserRole> builder) {
+    public void buildWith(final ConfigBuilder<CaseData, State, UserRole> configBuilder) {
 
-        builder
+        configBuilder
             .event(DRAFT_CREATE.name)
             .initialState(DRAFT)
             .name("Create draft case")
             .description("Apply for a divorce or dissolution")
             .displayOrder(1)
-            .postState(DRAFT.getValue())
             .retries(120, 120)
             .grant("CRU", CITIZEN)
-            .grant("CRU", CASEWORKER_DIVORCE_COURTADMIN_BETA)
-            .grant("CRU", CASEWORKER_DIVORCE_COURTADMIN)
-            .grant("CRU", CASEWORKER_DIVORCE_SOLICITOR)
-            .grant("CRU", CASEWORKER_DIVORCE_SUPERUSER)
-            .grant("CRU", CASEWORKER_DIVORCE_COURTADMIN_LA)
-            .fields()
-            .optional(CaseData::getCreatedDate)
-            .optional(CaseData::getD8caseReference)
-            .optional(CaseData::getD8legalProcess);
+            .grant("R",
+                CASEWORKER_DIVORCE_COURTADMIN_BETA,
+                CASEWORKER_DIVORCE_COURTADMIN,
+                CASEWORKER_DIVORCE_SOLICITOR,
+                CASEWORKER_DIVORCE_SUPERUSER,
+                CASEWORKER_DIVORCE_COURTADMIN_LA);
+
+        configBuilder
+            .workBasketInputFields()
```

Comment:
can we chain his about the fields section?

Reply:
Sadly not, as the workBasketInputFields method does not apply to the Event.Builder that is returned from the event method above